### ISSUE TITLE
[FEATURE] Cache labeling result to avoid unnecessary redraws
when refreshing canvas

### ISSUE THREAD

**Issue by QGIS-feature-tracker** [qgis/QGIS-Documentation#1672]
Original commit: https://github.com/qgis/QGIS/commit/33eb4bc0c4b047d461c374cf7bf1b361f2402f91 by nyalldawson

This change allows the labeling results to be cached to an image
following a map render. If the cached label result image can be
reused for the next render then it will be, avoiding the need
to redraw all layers participating in the labeling problem and
resolving the labeling solution.

Basically this means that canvas refreshes as a result of changes
to any NON-LABELED layer are much faster. (Changing a layer which
is part of the labeling solution still requires all labeled
layers to be completely redrawn)

**Comment by DelazJ** [comment]
Not sure this information belongs to user manual

**Comment by yjacolin** [comment]
+1 to close it

**Comment by DelazJ** [comment]
Thanks.
@yjacolin can you also give a look to some of the issues tagged [ToDocOrNotToDoc?](https://github.com/qgis/QGIS-Documentation/labels/ToDocOrNotToDoc%3F) and let me know those you agree we should close without writing about, please? Thanks.
